cartographer_ros/cartographer_ros/map_builder_bridge.cc

```diff
@@ -209,7 +209,7 @@ visualization_msgs::MarkerArray MapBuilderBridge::GetTrajectoryNodesList() {
     marker.header.frame_id = node_options_.map_frame;
     marker.color.b = 1.0;
     marker.color.a = 1.0;
```

Comment:
i.e. this :) http://martin.ankerl.com/2009/12/09/how-to-create-random-colors-programmatically